samples/bluetooth/direct_test_mode/src/fem/fem.c

```diff
@@ -319,3 +320,41 @@ int fem_init(NRF_TIMER_Type *timer_instance, uint8_t compare_channel_mask)
 
 	return 0;
 }
+
+#if defined(NRF52840_XXAA) || defined(NRF52833_XXAA)
+void fem_errata_254(nrf_radio_mode_t mode)
+{
+	static uint8_t old_mode = NRF_RADIO_MODE_NRF_1MBIT;
```

Comment:
Yes because different workaround is used for switching to ieee802.15.4 and different is when switching from this mode, so the old mode needs to be saved.